src/clients/FlexbaseClient.Card.ts

```diff
@@ -116,4 +116,20 @@ export class FlexbaseClientCard extends FlexbaseClientBase {
         return { success: false, error: 'Unable to update the card info', card: null };
       }
     }
+
+    async updateCardStatus(cardId: string, status: 'active' | 'suspended' | 'terminated', last4?: string): Promise<CardResponse> {
+      try {
+        const response = await this.client.url('/card/status').put({cardId, status, last4}).json<CardResponse>();
+
+        if (!response.success) {
+          this.logger.error('Unable to update the card status');
+          return response;
```

Comment:
If we're still returning the response regardless of the `success` value, then this return statement can be removed since we're already returning the response outside of the check.

Reply:
Oh yes, you're right. Removing this check.